src/SpotifyApi.NetCore/FollowApi.cs

```diff
@@ -102,5 +102,44 @@ public FollowApi(HttpClient httpClient) : base(httpClient)
             return await GetModel<T>(builder.Uri, accessToken);
         }
         #endregion
+
+        #region CheckCurrentUserFollowsPlaylist
+        /// <summary>
+        /// Check if Current User Follows Playlist
```

Comment:
Change to "Check to see if one or more Spotify users are following a specified playlist." as per Spotify docs.